lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -336,11 +400,13 @@ class AwsCompileEventBridgeEvents {
       Id: makeEventBusTargetId(RuleName),
     };
 
-    const target = this.addInputConfigToTarget({
+    const target = this.mountTargetConfig({
```

Comment:
What does `mount` mean in this context? It's not obvious to me but maybe I'm missing something here. Maybe we could just call it `configureTarget` ?